charts/prefect-server/values.yaml

```diff
@@ -31,12 +31,6 @@ global:
       # -- prefect image pull secrets
       pullSecrets: []
 
-    # -- sets PREFECT_API_URL
-    prefectApiUrl: http://localhost:4200/api
-
-    # -- sets PREFECT_SERVER_API_HOST
-    prefectApiHost: 0.0.0.0
```

Comment:
these are both redundant because:
- PREFECT_SERVER_API_HOST it is only used to construct the PREFECT_UI_API_URL if not provided
- PREFECT_API_URL is primarily used by the client, and is only used server side if PREFECT_UI_API_URL is not set

Reply:
With these removed, maybe we can add a template to [_validation.tpl](https://github.com/PrefectHQ/prefect-helm/blob/main/prefect-server/templates/_validation.tpl) to return an error/warning if the user tries to configure these removed values.